tools/SeeDot/faceDetection.md

```diff
@@ -0,0 +1,114 @@
+# Quantizing Face Detection using SeeDot
+
+Face Detection using SeeDot can be performed on the `face-2` and `face-4` datasets, which are processed subsets of the [SCUT Heads](https://github.com/HCIILAB/SCUT-HEAD-Dataset-Release) dataset.
+
+Face Detection is a regression problem that involves an image input with (or without) faces and bounding boxes around the faces as output. 
+Face Detection is supported for x86 and M3 target devices.  
+
+### Obtaining the datasets
```

Comment:
Let's keep the headings capitalized with the first letter of the words.

Reply:
This is still unchanged.

Reply:
My bad